src/components/Misc/PostButtonModal.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { Dispatch, ReactElement, SetStateAction } from "react";
+import { Dialog, DialogTitle, DialogContent, Button } from "@mui/material/";
+
+const styles = {
+  dialogPosition: {
+    position: "absolute",
+    bottom: "500px",
+  },
+};
+
+type ModalProps = {
+  children: ReactElement<any, any>;
+  openModal: boolean;
+  setOpenModal: Dispatch<SetStateAction<boolean>>;
+};
+
+export default function PostButtonModal(props: ModalProps) {
+  const { children, openModal, setOpenModal } = props;
+  return (
+    <Dialog
+      open={openModal}
+      fullWidth
+      scroll="paper"
+      sx={styles.dialogPosition}
+      PaperProps={{
```

Comment:
Do we need this? What does it do?

Reply:
**fullWidth**: Uses 100% of the parent's width

**scroll**: Has two options, ``paper`` and ``body``. 

> ``paper`` allows you to scroll through the ``DialogContent`` whilst fixing the location of the ``DialogTitle``
> ``body`` allows you to scroll through the ``DialogContent`` but this time you are scrolling through the entire body meaning the ``DialogTitle`` can be scrolled past.

**sx** and **PaperProps**: PaperProps is not necessary so I've removed it and decided to make a design choice for the positioning of the component. This component will now be placed in the middle of the page rather than closer to the top of the page.

Reply:
Talked about this offline, we'll create a new tech debt ticket to handle some of the styling details